### DISCUSSION TITLE
sw-text-editor component

### DISCUSSION THREAD

**Discussion by wannevancamp** [shopware/meteor-component-library#65] [Ideas]
Is there any plan to implement the sw-text-editor as a separate component? This would be very useful when developing components in the admin for the app system.

Example:
![image](https://github.com/shopware/meteor-component-library/assets/3399877/174d3535-abb7-4587-9bde-cc18f5e7511a)

**Comment by Weltraumakustik** [comment]
Hello @wannevancamp,

The WYSIWYG editor is on our list of components we'd like to add to the component library. Unfortunately, we cannot share any ETA just yet. We'll keep you updated!

**Reply by wannevancamp** [reply]
Thanks for the feedback!
